cityofaustin/atd-moped
# adds moped_user_saved_views_to_database

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/19686

## Testing
**URL to test:** 

local

**Steps to test:**
- restart cluster or apply migrations and metadata
- check to see that the `moped_user_saved_views` table exists and that the columns and permissions look correct
- check for the `set_moped_user_saved_views_updated_at` trigger function and the activity log transformation

---
#### Ship list
